index.html

```diff
@@ -3,14 +3,15 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Public Gateway Checker | IPFS</title>
+	<meta name="description" content="Checks which public IPFS gateways are online or not">
```

Comment:
@jessicaschilling I sneaked this to improve SEO. Right now it looks really messy:
> ![2020-10-20--22-35-08](https://user-images.githubusercontent.com/157609/96641185-a4d36580-1324-11eb-97a9-0320361b248c.png)